pkg/snapshot/restorer/restorer.go

```diff
@@ -189,11 +205,11 @@ func (r *Restorer) makeDB(snapdir string, snap *brtypes.Snapshot, commit int, sk
 	}
 	defer rc.Close()
 
-	if err := fileutil.CreateDirAll(snapdir); err != nil {
+	if err := fileutil.CreateDirAll(snapDir); err != nil {
```

Comment:
I touched this line only for the variable name change. Again, this code comes from the [original PR](https://github.com/gardener/etcd-backup-restore/pull/6) that introduces restoration feature, so I haven't really touched it all this while.

From the documentation, `fileutil.CreateDirAll` is similar to `os.MkDirAll` and uses `0700` access mode by default, and returns error is the deepest dir is not empty. In our case, this check is quite useless because restoration is performed only if the etcd directory does not exist already. So I see no harm in changing this to `os.MkDirAll` for the sake of consistency across the package. Thanks